core_integrations/asus_router

```diff
@@ -0,0 +1 @@
+asuswrt
```

Comment:
These are not files,  ou need to create a symbolic link.

Reply:
Sorry. Seems like I did not think a lot. I hope, now this is fine